azure-data-lake/1.0/modules/ROOT/pages/azure-data-lake-connector-reference.adoc

```diff
@@ -0,0 +1,1185 @@
+| Response Timeout a| Number |  The timeout for requests to the remote service. |  | 
+| Response Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the Response Timeout |  | 
+| Streaming Type a| Enumeration, one of:
+
+** AUTO
+** ALWAYS
+** NEVER |  Defines whether to send the request using streaming. Setting the value to `AUTO` will automatically define the best strategy based on the request content. |  | 
+| Target Variable a| String | Name of the variable in which to store the operation's output. |  | 
+| Target Value a| String |  An expression that evaluates the operation's output. The expression outcome is stored in the *Target variable*. |  &#35;[payload] | 
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  A retry strategy in case of connectivity errors. |  | 
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+| *Type* a| String
+| *Attributes Type* a| <<HttpResponseAttributes>>
+|===
+
+=== For Configurations
+
+* <<Config>> 
+
+=== Throws
+
+* AZURE-DATA-LAKE-STORAGE:BAD_REQUEST 
+* AZURE-DATA-LAKE-STORAGE:CLIENT_ERROR 
+* AZURE-DATA-LAKE-STORAGE:CONNECTIVITY 
+* AZURE-DATA-LAKE-STORAGE:INTERNAL_SERVER_ERROR 
+* AZURE-DATA-LAKE-STORAGE:NOT_ACCEPTABLE 
+* AZURE-DATA-LAKE-STORAGE:NOT_FOUND 
+* AZURE-DATA-LAKE-STORAGE:RETRY_EXHAUSTED 
+* AZURE-DATA-LAKE-STORAGE:SERVER_ERROR 
+* AZURE-DATA-LAKE-STORAGE:SERVICE_UNAVAILABLE 
+* AZURE-DATA-LAKE-STORAGE:TIMEOUT 
+* AZURE-DATA-LAKE-STORAGE:TOO_MANY_REQUESTS 
+* AZURE-DATA-LAKE-STORAGE:UNAUTHORIZED 
+* AZURE-DATA-LAKE-STORAGE:UNSUPPORTED_MEDIA_TYPE 
+
+
+[[CreateOrRename]]
+== Create/Rename Path
+`<azure-data-lake-storage:create-or-rename>`
+
+Creates or renames a file or directory path on the provided file system. 
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | *x*
+| File System a| String |  Target file system. |  | *x*
+| Path a| String |  Target path. |  | *x*
+| Resource a| String |  Only required for creation. The value must be `file` or `directory`. |  | 
+| Continuation a| String |  Optional. When renaming a directory, the number of paths that are renamed with each invocation is limited. |  | 
+| Mode a| String |  Optional. Valid only when namespace is enabled. This parameter determines the behavior of the rename operation. The value must be `legacy` or `posix`, and the default value is `posix`. |  | 
+| X-MS-Rename Source a| String |  An optional file or directory to rename. The value must have the following format:'/{filesystem}/{path}'. If "x-ms-properties" is specified, the properties overwrite the existing properties; otherwise, the existing properties are preserved. This value must be a URL percent-encoded string. Note that the string can contain only ASCII characters in the ISO-8859-1. |  | 
```

Comment:
Check the preview for this description. It doesn't look like it rendered correctly.